shared/Microsoft.Azure.Devices.Shared/TwinJsonConverter.cs

```diff
@@ -119,7 +119,14 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
                 writer.WritePropertyName(X509ThumbprintTag);
                 serializer.Serialize(writer, twin.X509Thumbprint);
             }
-
+ 
+#if ENABLE_MODULES_SDK
```

Comment:
Nit: I think it's OK to remove the conditionals now.

Reply:
I tried, but if we do that then we have to go into those regions and add the other #ifdefs (WINDOWS_UWP, PCL, NETMF) that we avoid currently because we simply don't define ENABLE_MODULES_SDK in those projects.

Reply:
Doesn't have to be in this check-in but is required before we merge into master.

Reply:
Agreed